spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/data/TestSparkParquetReader.java

```diff
@@ -144,16 +143,16 @@ public void testInt96TimestampProducedBySparkIsReadCorrectly() throws IOExceptio
             });
     List<InternalRow> rows = Lists.newArrayList(RandomData.generateSpark(schema, 10, 0L));
 
-    try (FileAppender<InternalRow> writer =
-        new ParquetWriteAdapter<>(
-            new NativeSparkWriterBuilder(outputFile)
-                .set("org.apache.spark.sql.parquet.row.attributes", sparkSchema.json())
-                .set("spark.sql.parquet.writeLegacyFormat", "false")
-                .set("spark.sql.parquet.outputTimestampType", "INT96")
-                .set("spark.sql.parquet.fieldId.write.enabled", "true")
-                .build(),
-            MetricsConfig.getDefault())) {
-      writer.addAll(rows);
+    try (ParquetWriter<InternalRow> writer =
```

Comment:
Why were these changes needed?

Reply:
this was using the deprecated `ParquetWriteAdapter` class, so I started changing code that uses this class so that we can eventually remove `ParquetWriteAdapter`